Assets/Scripts/Battle/BattleSystem/BattleParticipant.cs

```diff
@@ -18,10 +29,42 @@ public virtual void Initialize()
 
     public abstract void ChooseAction();
 
-    public void DealDamage(int damage)
+    /// <summary>
+    /// Handles when a Battle Participant attacks another battle participant.
+    /// </summary>
+    /// <param name="damage"></param>
+    /// <param name="attackElement"></param>
+    public void DealDamage(int damage, ElementType attackElement)
+    {
+        if (Weaknesses.Contains(attackElement))
+        {
+            damage *= 2;
+        }
+        else if (Resistances.Contains(attackElement))
+        {
+            damage /= 2;
+            Mathf.FloorToInt(damage);
```

Comment:
This line doesn't do anything. The damage is an integer, so it already uses integer division.